crates/pink/macro/src/macro_xcall.rs

```diff
@@ -367,15 +355,6 @@ fn patch_call_trait(mut input: syn::ItemTrait) -> syn::ItemTrait {
         if let syn::TraitItem::Method(method) = item {
             // Remove the call attribute
             method.attrs.retain(|attr| !attr.is_xcall());
```

Comment:
Why removed?

Reply:
It is because all of the actual methods this macro decorates already have `&mut self` and the llvm-cov report complain this branch of code never reached.